examples/sample-app/application-template-custombuild.json

```diff
@@ -66,6 +66,7 @@
       "parameters": {
         "output": {
           "to": {
+            "kind": "ImageStream",
```

Comment:
It's not, we should then, analogically to what we have in From, where ImageStreamTag is the default (see [here](https://github.com/openshift/origin/blob/master/pkg/build/api/v1/conversion.go#L16-L30)).